apps/site/pages/components/molecules/navbar-slider.mdx

```diff
@@ -0,0 +1,177 @@
+---
+description: As part of the Navbar, this component it its mobile version used to list the navigation links.
+---
+
+import path from 'path'
+import { useSSG } from 'nextra/ssg'
+import { Tab, Tabs } from 'nextra-theme-docs'
+import PropsSection from 'site/components/PropsSection'
+import { TokenTable, TokenRow } from 'site/components/Tokens'
+import { OverviewSection } from 'site/components/OverviewSection'
+import NavbarSliderUsage from 'site/components/Navigation/NavbarSliderUsage'
+import { Navbar, NavbarSlider, UIProvider } from '@faststore/ui'
+import { getComponentPropsFrom } from 'site/components/utilities/propsSection'
+
+export const getStaticProps = () => {
+  const navbarSliderPath = path.resolve(__filename)
+  const components = ['NavbarSlider.tsx']
+  const [navbarSliderProps] = getComponentPropsFrom(
+    navbarSliderPath,
+    components
+  )
+  return {
+    props: {
+      // We add an `ssg` field to the page props,
+      // which will be provided to the Nextra `useSSG` hook.
+      ssg: {
+        navbarSliderProps,
+      },
+    },
+  }
+}
+
+export const NavbarSliderPropsSection = ({ component }) => {
+  // Get the data from SSG, and render it as a component.
+  const { navbarSliderProps } = useSSG()
+  return {
+    NavbarSlider: <PropsSection propsList={navbarSliderProps} />,
+  }[component]
+}
+
+<header>
+
+# Navbar Slider
+
+As part of the [Navbar](/components/organisms/navbar), this component is its **mobile version** used to list the navigation links (through the [NavbarLinks](/components/molecules/navbar-links)).
+
+This component is part of the [Navigation](/components/features/navigation) feature and must be used in the [Navbar](/components/organisms/navbar) context.
+
+</header>
+
+## Import
+
+Import the component from [@faststore/ui](/../components)
+
+```tsx
+import { NavbarSlider } from '@faststore/ui'
+```
+
+Import Styles
+
+```tsx
+import '@faststore/ui/src/components/molecules/NavbarSlider/styles.scss'
+```
+
+---
+
+## Usage
+
+**Note:** This component is intended to be used only in the **mobile version**, but it can be adapted to a desktop version by customizing the component styles.
```

Comment:
How about using a Callout component here?

```suggestion
<Callout type="warning" emoji="⚠️">
This component is intended to be used only in the **mobile version**, but it can be adapted to a desktop version by customizing the component styles.</Callout>
```